src/generateSdk/sdkGenerator/PythonSdkGenerator.ts

```diff
@@ -176,7 +202,118 @@ class SdkGenerator implements SdkGeneratorInterface {
       }
 
       for (const externalType of externalTypes) {
-        view.externalTypes.push({type: this.generateExternalType(externalType)});
+        if (externalType.path && !classInfo.classConfiguration.path.includes(externalType.path)) {
+          let currentView = null;
+          for (const nestedExternalType of externalTypes) {
+            const isUsed = this.isExternalTypeUsed(externalType, nestedExternalType);
```

Comment:
`isExternalTypeUsed` change name to reflect what is doing, maybe: `isExternalTypeUsedBy`?